### PR TITLE
docs: document processor_output usage in VLM offline engine notebook

### PR DESCRIPTION
**Pull request by minleminzui**
## Motivation

Update vlm_query.ipynb to include a Qwen2.5-VL example that passes
HuggingFace processor_output into Engine.generate, aligning the docs
with the three supported VLM input formats (basic, processor_output,
precomputed_embedding).

#10532

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: docs/advanced_features/vlm_query.ipynb [hunk 1]
```diff
@@ -106,6 +106,38 @@
...
+    "\n",
+    "processor = AutoProcessor.from_pretrained(model_path, use_fast=True)\n",
+    "processor_output = processor(\n",
+    "    images=[image], text=conv.get_prompt(), return_tensors=\"pt\"\n",
+    ")\n",
+    "\n",
+    "out = llm.generate(\n",
+    "    input_ids=processor_output[\"input_ids\"][0].detach().cpu().tolist(),\n",
+    "    image_data=[dict(processor_output, format=\"processor_output\")],\n",
+    ")\n",
```
[the remark this answers is not in the document]
Comment (minleminzui) [reply docs/advanced_features/vlm_query.ipynb:132]: For the offline Engine.generate API we intentionally require the caller to provide either a prompt or explicit input_ids.

The image_data argument is only used to carry the visual side (raw images / processor_output / precomputed embeddings). The engine does not implicitly extract input_ids from a nested processor_output dict – we want to keep the contract simple and avoid tying the engine to the internal structure of a specific HF processor.

In the current implementation this call:
```python
out = llm.generate(
    input_ids=processor_output["input_ids"][0].detach().cpu().tolist(),
    image_data=[dict(processor_output, format="processor_output")],
)

```
is the correct usage and matches what we test in test_vlm_input_format.py.

We can consider adding a higher-level convenience wrapper in the future, but for now I’d prefer to keep the docs consistent with the actual engine API.
